dbgpt/app/knowledge/service.py

```diff
@@ -79,6 +80,12 @@ def create_knowledge_space(self, request: KnowledgeSpaceRequest):
         )
         if request.vector_type == "VectorStore":
             request.vector_type = CFG.VECTOR_STORE_TYPE
+        if request.vector_type == "KnowledgeGraph":
+            knowledge_space_name_pattern = r"^[a-zA-Z0-9\u4e00-\u9fa5]+$"
+            if re.match(knowledge_space_name_pattern, request.name):
+                return True
```

Comment:
Why `return` here?  the process will be exit and space will not be created.    Maybe it should be
```
knowledge_space_name_pattern = r"^[a-zA-Z0-9\u4e00-\u9fa5]+$"
if !re.match(knowledge_space_name_pattern, request.name):
      raise Exception(f"space name:{request.name} invalid")
```

Reply:
Thanks. I made a mistake here.
I fixed it in the newest commit.